.circleci/config.yml

```diff
@@ -66,3 +72,29 @@ jobs:
           paths:
             - ~/.gradle
           key: gradle-repo-v1-{{ .Branch }}-{{ checksum "build.gradle" }}
+
+  snyk:
+    docker:
+      - image: cimg/openjdk:8.0
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          key: v1-gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}
+      - restore_cache:
+          keys:
+            # when lock file changes, use increasingly general patterns to restore cache
+            - gradle-repo-v1-{{ .Branch }}-{{ checksum "build.gradle" }}
+            - gradle-repo-v1-{{ .Branch }}-
+            - gradle-repo-v1-
+
+      - run:
+          name: Snyk 2.0
+          command: ./gradlew snyk-test snyk-monitor
+
+      - run:
+          name: Snyk legacy
+          command: cd symphony-bdk-legacy && ./gradlew snyk-test snyk-monitor
```

Comment:
can we use `./gradlew -p symphony-bdk-legacy snyk-test snyk-monitor` instead of cd?

Reply:
I changed it because I noticed the snyk-test plugin was not taking into account the -p option but was running snyk in the current folder